pavelib/assets.py

```diff
@@ -384,14 +384,25 @@ def coffeescript_files():
 
 @task
 @no_help
-def compile_coffeescript(*files):
+def compile_coffeescript(*coffee_files):
```

Comment:
Can you add documentation to this? Is `coffee_files` a list of files, or a bash command to get a list of files? In the former case, `js_find_cmd` is broken, because the format command it uses only has space for a single value.